tools/harness-thci/README.md

```diff
@@ -28,3 +29,28 @@ new THCI implementation.
 6. Drag OpenThread: TI CC2538DK reference device to Test Bed list with desired number.
 
 7. Select one or multiple test cases to execute.
+
+
+## OpenThread WpanCtl Environment Setup ##
+
+1. Copy the "OpenThread.png" to \GRL\Thread1.1\Web\images folder.
```

Comment:
Remove "the" in bullets 1, 2, and 3, or add "file" to the sentence.

So either:
Copy "OpenThread.png" to...
Copy "deviceInputFields.xml" to...
Copy "OpenThread_WpanCtl.py" to...

Or:
Copy the "OpenThread.png" file to...
Copy the "deviceInputFields.xml" file to
Copy the "OpenThread_WpanCtl.py" file to